content/blog/2024/12/2024-12-19-matrix-v1.13-release.md

```diff
@@ -0,0 +1,138 @@
++++
+title = "Matrix v1.13 release"
+date = "2024-12-19T19:53:43Z"
+path = "/blog/2024/12/19/matrix-v1.13-release"
+
+[taxonomies]
+author = ["Travis Ralston"]
+category = ["Releases", "Spec"]
++++
+
+Hey all,
+
+Another 9 MSCs have been released today in [Matrix 1.13](https://spec.matrix.org/v1.13/)! It’s just over 2 months since [Matrix 1.12](https://matrix.org/blog/2024/10/07/matrix-v1.12-release/) went out, and the last scheduled release for 2024 - the next release is planned for around FOSDEM 2025. Today’s release contains more T&S features and a number of clarifications and improvements. The full changelog is at the end of this post, per usual :)
+
+
+## Account suspension
+
+Last release brought [Account Locking](https://spec.matrix.org/v1.13/client-server-api/#account-locking)* *to the spec, and this release brings [Account Suspension](https://spec.matrix.org/v1.13/client-server-api/#account-suspension) - a highly related but semantically different feature. Locking prevents the user from accessing their account, typically as a consequence of security reasons or restrictions, while suspension aims to disrupt while retaining a (mostly) readonly experience for the user. Both actions are reversible, and available to safety and security teams to help manage their servers with alternatives to deactivation (a destructive, irreversible, action).
```

Comment:
The first link has weird *s after it. Did formatting get lost?

Reply:
thanks! 1ae1c25ee94dc320f4dc97790a484723562645b1